apps/design/backend/package.json

```diff
@@ -21,7 +19,12 @@
     "clean:self": "rm -rf build && tsc --build --clean tsconfig.build.json",
     "create-org": "node -r esbuild-runner/register ./scripts/create_org.ts",
     "create-user": "node -r esbuild-runner/register ./scripts/create_user.ts",
-    "db:reset": "./scripts/db_reset_dev.sh",
+    "db:migrations:create": "node-pg-migrate create",
+    "db:migrations:run": "node-pg-migrate up",
+    "db:migrations:run-dev": "DATABASE_URL='postgresql://design:design@localhost:5432/design' node-pg-migrate up",
```

Comment:
I wonder if we could combine each of these command pairs into a single command that uses the env's `DATABASE_URL` if set, otherwise uses the default dev db URL. Might just simplify usage a tad

Reply:
Cool, explicitness is a good enough reason for me!